src/components/Common/BaseContextMenu.tsx

```diff
@@ -0,0 +1,175 @@
+import { IconDotsVertical } from '@tabler/icons-react';
+import { Fragment, useMemo } from 'react';
+
+import { useTranslation } from 'next-i18next';
+
+import classNames from 'classnames';
+
+import { getByHighlightColor } from '@/src/utils/app/folders';
+
+import {
+  BaseContextMenuProps,
+  BaseMenuItemRendererProps,
+} from '@/src/types/menu';
+
+import { Menu, MenuItem } from './DropdownMenu';
+import { Tooltip, TooltipContent, TooltipTrigger } from './Tooltip';
+
+function BaseContextMenuItemRenderer({
```

Comment:
Do we really need base? Just context menu... is not enough?

Reply:
1. we already have `ContextMenu` component
2. it's base component for all other context menus